mock-app-verifier/scripts/createRustVerifyingKey.ts

```diff
@@ -1,2 +1,9 @@
-import { createVerifyingkeyRsFile } from "../../light-zk.js/src/cli-utils/createRustVerifyingKey";
-createVerifyingkeyRsFile();
+import { createVerfyingkeyRsFileArgv } from "../../cli/src/utils/createRustVerifyingKey";
```

Comment:
I want to check whether this is used

Reply:
the "removed" ones were addressed to an invalid path.
the ```createVerifyingkeyRsFile``` has now 6 input args.
the most natural way to fix it was to replace with the Argv-based variant.

but please, let's talk about

Reply:
it's not use the file can be deleted

Reply:
👌